### PR TITLE
Updated database readme : Caleb

### LINKED ISSUE #201: Update database readme
As a developer, I want clear documentation that outlines the installation process, the structure of the pipeline, and how to run the app, so new contributors can easily get started and understand the functionality of the app.

- Edit README for database setup so that it is up to date for the project

### PR DESCRIPTION
**Pull request by CalebMathers**
This closes #201 

Updated the database readme to fill out the empty sections and ensure it is up to date.

### CODE REVIEW

File: database/README.md [hunk 1]
```diff
@@ -6,7 +6,59 @@ This folder contains the scripts to create the database, database tables(as show
...
+- To get the current count of the total rows in each of the dynamic tables excluding subscriber related tables (solar_feature, aurora_alert, image, forecast, body_assignment):
+```bash count.sh```
 
-## How it works
\ No newline at end of file
+## How it works
+#### `fake_data.sql`
+- Uses SQL DML to fill the dynamic tables with fake data for testing purposes.
+#### `schema.sql`
+- Uses SQL DDL to create all the tables in the database as per the ERD above. Then uses SQL DML to seed the static data.
```
Comment (AlexOBarnes) [inline database/README.md:52]: Very consistent

**Review by AlexOBarnes — APPROVED** [review]

File: database/README.md [hunk 2]
```diff
@@ -6,7 +6,59 @@ This folder contains the scripts to create the database, database tables(as show
 __Figure 1__ - _Starwatch Database ERD_: Shows the entity relationship diagram for the database used in this project.
 
 ## Setup
```
Comment (MurasatoNaoya) [inline database/README.md:8]: Clear, consistent formatting using markdown.

File: database/README.md [hunk 3]
```diff
@@ -6,7 +6,59 @@ This folder contains the scripts to create the database, database tables(as show
...
+
+2. Create a `.env` file with the following:
+    - DB_HOST - to access your RDS instance
+    - DB_USER - username for accessing RDS
+    - DB_PASSWORD - password for accessing RDS
+    - DB_NAME - the name of your database
+    - DB_PORT - port to access RDS (typically 5432 for postgres)
 
 ## Usage
+- To create the initial tables and seed the static data tables (aurora_colour, region, county, body, and constellation):
```
Comment (MurasatoNaoya) [inline database/README.md:33]: Clear explanation about what each shell script is for and how to run them.

File: database/README.md [hunk 4]
```diff
@@ -6,7 +6,59 @@ This folder contains the scripts to create the database, database tables(as show
 __Figure 1__ - _Starwatch Database ERD_: Shows the entity relationship diagram for the database used in this project.
 
 ## Setup
+### To Run Locally:
```
Comment (MurasatoNaoya) [inline database/README.md:9]: Extensive explanation on how to run a similar RDS database in a similar fashion to StarWatch.

**Review by MurasatoNaoya — APPROVED** [review]
LGTM.

**Pull request merged**